PYTHIA6/AliPythia6/AliGenPythiaPlus.cxx

```diff
@@ -1381,81 +1339,365 @@ void AliGenPythiaPlus::GetSubEventTime()
   return;
 }
 
+///
+/// \return true if particle in central barrel acceptance
+///
+/// \param eta        particle pseudorapidity,  etamin=-etamax
+///
+Bool_t AliGenPythiaPlus::IsInBarrel(Float_t eta) const
+{
+  if( eta < fTriggerEta  ) 
+  return kTRUE;
+  else 
+  return kFALSE;
+}
```

Comment:
Thanks. So it seems that we have a lot of common functionality between the 2 classes. The proper thing would probably be to factor out common code or dispatch from one to the other.
[This is something we can keep in mind for AliceO2 ... not asking to completely change the classes now.]